spec/Section 3 -- Type System.md

```diff
@@ -1679,6 +1701,21 @@ is constructed with the following rules:
   variable definition does not provide a default value, the input object field
   definition's default value should be used.
 
+Further, if the input object is a _OneOf Input Object_, the following additional
+rules apply:
+
+- If the input object literal or unordered map does not contain exactly one
+  entry an _execution error_ must be raised.
+
+- Within the input object literal or unordered map, if the single entry is the
```

Comment:
same here?

Reply:
Copying [this comment from 2021](https://github.com/graphql/graphql-spec/pull/825#discussion_r588879692) here:

> These are already enforced by validation, so I do not know if we need them in coercion. However, if we do not enforce them here then the coercion table below would have different outputs (e.g. `{a: $a, b: $b}` with `{ "a": "abc" }` would be allowed by coercion, but not by validation).
> 
> An alternative approach is to keep the above and add a different non-null coerced value rule so we can drop the two rules below (since they'll be implicitly true).
> 
> @leebyron what are your thoughts on this?